miso-web/src/main/webapp/scripts/bulk_sample.js

```diff
@@ -1567,4 +1587,21 @@ BulkTarget.sample = (function ($) {
       return arr2.includes(x);
     });
   }
+
+  function updateProjectAssays(projectSelected, api, rowIndex) {
+    var notArchivedProjectAssays = [];
+    for (var i = 0; i < projectSelected.assayIds.length; i++) {
+      var tempAssay = Constants.assays.find(function (x) {
+        return x.id === projectSelected.assayIds[i];
+      });
+      if (!tempAssay.archived) {
+        notArchivedProjectAssays.push(tempAssay);
+      }
+    }
+    api.updateField(rowIndex, "requisitionAssayId", {
+      source: notArchivedProjectAssays,
+      value: null,
```

Comment:
Don't clear the value, as the user might not notice this. If it become invalid with the change, it will appear red